src/components/generics/vacation-test-forms/myVacationRequests.tsx

```diff
@@ -211,7 +282,7 @@ const renderVacationRequests = () => {
                   <TableCell>{ request.days }</TableCell>
                   <TableCell>{ request.startDate }</TableCell>
                   <TableCell>{ request.endDate }</TableCell>
-                  <TableCell>{ request.status }</TableCell>
+                  <StyledTableCell sx={{ "&.pending": { color: "#FF493C" }, "&.approved": { color: "#45cf36" } }} className={request.status === "APPROVED" ? "approved" : "pending"}>{ request.status }</StyledTableCell>
```

Comment:
This needs indentation fixing

Reply:
It also seems like there is an error in the VSCode regarding the scope prop within the DateRangePicker (around line 307) I jus can't comment on the line in this pull request for some reason.